content/articles/data-viz-chartify/index.md

```diff
@@ -0,0 +1,365 @@
+        numeric_column='generation')
+ch.show()
+```
+The first few arguments that we passed are `.set_title` and `.set_subtitle` which are for your chart title and subtitle. You can specify a name for your x and y axes using `.axes.set_xaxis_label` and `.axes.set_yaxis_label` respectively. The last argument, `.plot.bar` requires your data_frame which is the data to be plotted. The other parameters define the categorical and numerical columns you want to plot. 
+
+![Chartify](/engineering-education/data-viz-chartify/chartifytwo.png)
+
+*Screenshot by author*
+
+Next, let’s look at the pokemon type distribution but this time taking account of the ‘is_legendary’ attribute of each Pokemon. We would also start by creating a dataframe of the data needed. We can achieve this by grouping our Pokemon by the “type1” and “is_legendary” columns. This can be done by running the code snippet below:
+
+```python
+Pokemon_type_distribution = (pokemon_df.groupby(['type1','is_legendary'])['s/n'].sum().reset_index())
+print(Pokemon_type_distribution.head())
+```
+Then we would visualize this data with Chartify by running the code below:
+
+```python
+ch = chartify.Chart(blank_labels=True, x_axis_type='categorical')
+ch.set_title("Pokemon EDA using Chartify")
+ch.set_subtitle("Pokemon type distribution")
+ch.plot.bar(
+        data_frame=Pokemon_type_distribution,
+        categorical_columns=['type1', 'is_legendary'],
+        numeric_column='s/n', color_column='is_legendary')
+ch.show()
+```
+![Chartify](/engineering-education/data-viz-chartify/chartifythree.png)
+
+*Screenshot by author*
+
+### Number of Pokemon in each generation
+
+Let's take a look at the number of Pokemon in each generation. We would extract this data from the Pokemon data by running the code snippet below.
+
+```python
+generations = pd.DataFrame ({'count': pokemon_df.generation.value_counts().sort_index()})
+generations['generation'] = generations.index.get_level_values(0)
+generations
+```
+
+Next, we would run a code snippet like the one we ran for the previous bar chart.
+
+```python
+ch = chartify.Chart(blank_labels=True, x_axis_type='categorical')
+ch.set_title("Pokemon EDA using Chartify")
+ch.set_subtitle("Number of Pokemon in Each Generation")
+ch.plot.bar(data_frame=generations, categorical_columns='generation', 
+                             numeric_column='count',color_column='generation')
+ch.style.color_palette.reset_palette_order()
+ch.plot.text(
+        data_frame=generations,
+        categorical_columns='generation',
+        numeric_column='count',
+        text_column='count',
+        color_column='generation')
+ch.show()
+```
+We did not discuss the `color_column` in the previous bar chart. The `color_column` parameter gives your plot a color-based argument that colors the plot of the basis of the column specified. In our graph, Chartify will assign different colors to the different generations because we set `color_column` to be the column “generation”.
```

Comment:
```suggestion
The `color_column` parameter gives your plot a color-based argument that colors the plot of the basis of the column specified. In our graph, Chartify will assign different colors to the different generations because we set `color_column` to be the column `generation`.
```

Reply:
Remove this statement